website/docs/docs/collaborate/manage-access/audit-log.md

```diff
@@ -5,12 +5,15 @@ description: "You can troubleshoot possible issues and provide security audits b
 sidebar_label: "Audit log"
 ---
 
-To review actions performed by people in your organization, dbt provides logs of audited user and system events. The dbt Cloud audit log lists events triggered in your organization within the last 90 days.
-
-Use the audit log to quickly review the actions performed by members of your organization. The audit log includes details such as who performed the action, what the action was, and when it was performed. You can use these details to troubleshoot access issues, perform security audits, or analyze specific events.
+To review actions performed by people in your organization, dbt provides logs of audited user and system events. You can use the audit log to quickly review the actions performed by members of your organization. The audit log includes details such as who performed the action, what the action was, and when it was performed. You can use these details to troubleshoot access issues, perform security audits, or analyze specific events. 
 
 You must be an **Account Admin** to access the audit log and this feature is only available on Enterprise plans.
 
+The dbt Cloud audit log stores all the events that occurred in your organization:
+
+- For events within 90 days, the dbt Cloud audit log has a selectable date range that lists events triggered.
+- For events beyond 90 days, **Account Admins** can [export all events](#exporting-logs) by using **Export All**.
```

Comment:
Is there any limit to this feature, or will it be for the entire life of the account, regardless of age? The cloud offering may not be mature enough to document yet, but if there's an auto purge of this data after 3 or 5 years, or whatever, we should note it in the future.

Reply:
great question @matthewshaver! here's what Rob just replied below. there was no mention of the events going back to early 2022 originally so I'm not sure if there was a reason why that wasn't included in the first place. do you think i should add?

> we won't purge data from the 'export all' option, but it does only have events going back to jan or feb 2022, because we didn't start recording these events until then